tools/codegen/gen_functionalization_type.py

```diff
@@ -175,7 +182,7 @@ def emit_view_functionalization_body(
         at::AutoDispatchSkipFunctionalize guard;
         {meta_conversion_str}
         reference_tensor_output = at::_ops::{api_name}::call({', '.join(meta_call_args)});
-        tmp_output = at::_ops::{api_name}::redispatch({', '.join(view_redispatch_args)});
+        tmp_output = at::_ops::{api_name}::call({', '.join(view_redispatch_args)});
```

Comment:
This is subtle, probably worth a comment replicating what you said in your PR description